makeflow/src/makeflow.c

```diff
@@ -1599,6 +1626,19 @@ int main(int argc, char *argv[])
 		makeflow_gc_method = MAKEFLOW_GC_ALL;
 	}
 
+	if(wrapper_umbrella && wrapper_umbrella->spec) {
```

Comment:
After further discussion, not specifying the executable is fine, but if it is not specified we should emit a warning that umbrella will be used at the execution site.